Makefile

```diff
@@ -37,6 +38,7 @@ VERSION := $(shell git describe --tags --always --dirty)
 SRC_DIRS := cmd pkg # directories which hold app source (not vendored)
 
 ALL_ARCH := amd64 arm arm64 ppc64le
+ALL_PLATFORMS := linux-amd64 freebsd-amd64 freebsd-386
```

Comment:
this list should obsolete ALL_ARCH and should at least include all of those for linux.

Reply:
So the assume that containers should be built for linux-* in ALL_PLATFORMS?

Reply:
Hmm, I guess we need ALL_PLATFORMS to be different from container-able platforms, since they are not the same.

Maybe:

BUILD_PLATFORMS := a b c d e

CONTAINER_PLATFORMS := a b c  # must be a subset of BUILD_PLATFORMS